cfpb/consumerfinance.gov
# Archiving: sidebar content update

Remove language about 2 year timeframe from archived page sidebar content.


## Removals

- First sentence from archived page sidebar content

## How to test this PR

1. Check out branch
2. Archive a blog page or navigate to an archived blog page
3. Verify updated sidebar content


## Screenshots

<img width="1365" alt="Screen Shot 2021-02-18 at 10 12 05 AM" src="https://user-images.githubusercontent.com/778171/108402342-361eef80-71d2-11eb-9f2f-1966147f7b1c.png">
